### DISCUSSION TITLE
🖼️ ANY Experience Gallery

### DISCUSSION THREAD

**Discussion by fuksman** [anyproto/contributors#123] [Announcements]
Dear contributors,

We’re excited to introduce the [Gallery](https://gallery.any.coop/) – a way for you to share your ANY Experience with other users. We’re sure you’re proud of your setups for knowledge management, task tracking, studying approaches, and so on. This is your chance to share your setups with the community so they can get the most out of  Anytype!

To create and share your own ANY Experience, follow these steps (you will need a Github account for this):
1. Create a **new space** which includes _only_ the types, relations, widgets, templates, and objects you want to share. This will avoid accidental sharing of sensitive data from your primary space. Please note: the name and icon of this new space will be displayed for everyone who will install your ANY experience.
2. When the space is ready, export it in Any-Block JSON format. Click on the space name in the top-left corner, click “Export Space”, choose “Any-Block” option and set up following export settings:
    - File format – JSON,
    - Turn on “Zip archive”
    - Turn on “Include files”
3. Make one or several screenshots which show important features of your ANY experience. Save these screenshots in .png format.
4. Now, having created an exported space and screenshots, you can upload them to the Gallery. Explore existing Experiences in the [Gallery](https://gallery.any.coop/) to get examples of naming, descriptions, and screenshots. Then, please login to Github, follow [this link](https://github.com/anyproto/gallery/issues/new?assignees=&labels=experience&projects=&template=new_experience.yml) to create a request and fill the form:
    - Give a title to the request
    - Drag and drop exported Zip archive to the “Experience” field
    - Write a title for your experience in the “Title” field; it should be unique, pretty short, and written in English
    - Drag and drop .png screenshots to the “Screenshots” field
    - Write an English description for the experience in the “Description” field
    - Choose one or several categories which suit for your experience in the “Categories” field
    - If the content inside the Experience is not in English, please choose the language in the “Language” field
    - For now, all experiences are published under the MIT license; please confirm that you agree with it, that you have rights for posting such content, and that there is not harmful content in the experience in the “Terms” section
    - When ready, click “Submit new issue” button
5. Someone from Open Any Team will review your submission, and, when approved, the experience will be published in the [Gallery](https://gallery.any.coop/). You will be recognized as a contributor in our [Github](https://github.com/anyproto/contributors) and on our [website](https://anytype.io/contributors).

Excited to see your contributions,
Open Any Team

**Comment by amjarmed** [comment]
I am excited to share with you a new repository on GitHub that I've recently created – "[Awesome-AnyType-Resources Public](https://github.com/amjarino/Awesome-AnyType-Resources)."

**Comment by OrcVole** [comment]
Could we please have an example of a campaign website? For example, a "Save the Whale" site? This would showcase how anytype could be used to help co-ordinate activism, volunteers, subscribe to a calendar and donate and receive tasks, etc.
